data/assets/scene/solarsystem/heliosphere/todayssun/sunEarthLine.asset

```diff
@@ -0,0 +1,39 @@
+local earth = asset.require("scene/solarsystem/planets/earth/earth")
```

Comment:
The filename should be `sunearthline.asset` or `sun_earth_line.asset` since we only use lowercase letters for filenames